### PR TITLE
solidus_dev_support update

### PR DESCRIPTION
**Pull request by elia**
Update to the latest dev-support and make good use of the default configuration class, using it as the feed registry instead of using Feeds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
-# SolidusFeeds
+# Solidus Feeds
 
+[![CircleCI](https://circleci.com/gh/nebulab/solidus_feeds.svg?style=shield)](https://circleci.com/gh/nebulab/solidus_feeds)
+[![codecov](https://codecov.io/gh/nebulab/solidus_feeds/branch/master/graph/badge.svg)](https://codecov.io/gh/nebulab/solidus_feeds)
+
+<!-- Explain what your extension does. -->
```
Comment (igorbp) [inline README.md:6]: Did it slip in or is it here as a reminder to expand the description?

File: README.md [hunk 2]
```diff
@@ -13,20 +17,21 @@ gem 'solidus_feeds'
...
 
 ```shell
-bundle
-bundle exec rails g solidus_feeds:install
+bin/rails generate solidus_feeds:install
 ```
 
 ## Usage
 
+<!-- Explain how to use your extension once it's been installed. -->
```
Comment (igorbp) [inline README.md:25]: > Did it slip in or is it here as a reminder to expand the description?

Same question as line 6 🤔

**Review by igorbp — APPROVED** [review]
Nice! :D

File: README.md (hunk 2 above)
Comment (elia) [reply README.md:25]: That's part of the template offered by dev-support, leaving it there, given it's not visible in the readme, will make it easier to deal with diffs at the next update

**Review by nirebu — APPROVED** [review]

**Pull request merged**